geoutils/satimg.py

```diff
@@ -326,20 +321,15 @@ def __parse_metadata_from_fn(self,silent=False):
     def __parse_metadata_from_file(self,fn_meta):
         pass
 
-    def __copy__(self):
```

Comment:
I trust that removing the magic method doesn't have a weird side-effect anywhere? I think it is used for the syntax:
```python
r2 = copy(r)
```
which I don't think anybody uses ever, so it won't matter too much!

Reply:
All tests are passing, so shouldn't be any issue. 
They were not passing with the `__copy__` method in DEM and SatelliteImage and without one in Raster, it had weird side-effects (infinite recursions).

Reply:
Okay. We can just fix that later if it turns out to be a problem.